metrics/network/remote_network/remote-networking-iperf3.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This will test bandwidth using iperf3
```

Comment:
I wonder why is the reason about to have this test in another directory

Reply:
@MarioCarrilloA as this is using a remote setup with other configurations, I will like to separate local setup and remote setup

Reply:
besides more remote setup tests will be added